packages/commonwealth/client/scripts/hooks/useManageDocumentTitle.ts

```diff
@@ -0,0 +1,9 @@
+import { useEffect } from 'react';
+
+const useManageDocumentTitle = (title: string) => {
+  useEffect(() => {
+    document.title = title;
+  }, []);
```

Comment:
please fix the warning 
<img width="708" alt="image" src="https://github.com/hicommonwealth/commonwealth/assets/14819225/d64e4bb7-62ee-4432-a62b-d22819f68a72">

Reply:
Fixed.